integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonTableReader.java

```diff
@@ -314,7 +315,7 @@ private CarbonTable loadTableMetadata(SchemaTableName schemaTableName) {
     for (SchemaTableName table : tableList) {
       if (!table.equals(schemaTableName)) continue;
 
-      return parseCarbonMetadata(table);
+      return parseCarbonMetadata(table).carbonTable;
```

Comment:
why here need to add ".carbonTable"? 

Inside parseCarbonMetadata(table), already have the below code 👍 
result = cache.carbonTable;